src/server.js

```diff
@@ -220,11 +260,10 @@ app.get(
     const userData = Object.values(users);
 
     if (userData.length > 0) {
-      // Currently accessing the latitude and longitude of the first user (MVP).
-      // TODO (Chisom): Test average geolocation with multiple user locations.
-      const lat = userData[0].lat.toString();
-      const long = userData[0].long.toString();
-      const radiusMeters = "50000";
+      const averageLocation = averageGeolocation(userData);
```

Comment:
i could be getting my JS wrong, but could you do something like {latitude, longitude} = averageGeolocation(...)?

Reply:
I thought perhaps averageLocation would be a better description. But doing it this way would also be useful in understanding what exactly the key names are.

Reply:
done